assets/scss/local.scss

```diff
@@ -21,31 +21,35 @@
   width: 20em;
 }
 
-// Custom grid and border for goals section of overview page
+// Custom column with vertical border
 
-.govuk-grid {
-  gap: 20px;
-  display: flex;
+.app-table-column-with-right-border {
+  border-right: 1px solid $govuk-border-colour;
+  display: inline-block;
 }
 
-.govuk-grid-item {
-  padding: 16px;
-  text-align: center;
-}
+.overview-page-goal-summary-counts {
```

Comment:
I've put the govuk grid style overrides within a parent class. Otherwise it would mean that if we use govuk grids on a different page (for some other content/layout) they would also use these styles